charts/spire/charts/spire-server/templates/configmap.yaml

```
@@ -28,6 +28,16 @@ data:
         common_name = {{ .common_name | quote }},
         {{- end }}
       }
+      {{- with .Values.federation.bundleEndpoint }}
```

```suggestion

      {{- with .Values.federation.bundleEndpoint }}
```

add a new line to space it from the previous block